Makefile

```diff
@@ -0,0 +1,14 @@
+VERSION := `head -1 debian/changelog | awk '{print $2}' | grep -o '\d\+.\d\+'`
```

Comment:
My environment had an issue with this.  It did not get the version number.  Perhaps my version of grep?  This worked for me though (and supports versions with more then one dot):

      head -1 debian/changelog | awk '{print $2}' | grep -o -e '\([0-9\.]\+\)' | tr -d '()'